charts/kube-otel-stack/values.yaml

```diff
@@ -244,9 +251,191 @@ metricsCollector:
+            timestamp:
+              parse_from: time
+              layout: "%Y-%m-%dT%H:%M:%S.%LZ"
+          # Parse Docker format
+          - type: json_parser
+            id: parser-docker
+            output: extract_metadata_from_filepath
+            timestamp:
+              parse_from: time
+              layout: "%Y-%m-%dT%H:%M:%S.%LZ"
+          # Extract metadata from file path
+          - type: regex_parser
+            id: extract_metadata_from_filepath
+            regex: '^.*\/(?P<namespace>[^_]+)_(?P<pod_name>[^_]+)_(?P<uid>[a-f0-9\-]{36})\/(?P<container_name>[^\._]+)\/(?P<run_id>\d+)\.log$'
+            parse_from: $$attributes.file_path
+          # Move out attributes to Attributes
+          - type: metadata
+            attributes:
+              stream: "EXPR($.stream)"
+              k8s.container.name: "EXPR($.container_name)"
+              k8s.namespace.name: "EXPR($.namespace)"
+              k8s.pod.name: "EXPR($.pod_name)"
+              run_id: "EXPR($.run_id)"
+              k8s.pod.uid: "EXPR($.uid)"
+          # Clean up log record
+          - type: restructure
+            id: clean-up-log-record
+            ops:
+              - remove: logtag
+              - remove: stream
+              - remove: container_name
+              - remove: namespace
+              - remove: pod_name
+              - remove: run_id
+              - remove: uid
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 75
+        spike_limit_percentage: 30
+      resourcedetection/env:
+        detectors: [env]
+        timeout: 2s
+        override: false
+      batch:
+        send_batch_size: 1000
+        timeout: 1s
+        send_batch_max_size: 1500
+      k8sattributes:
+        passthrough: false
+        pod_association:
+          - sources:
+              - from: resource_attribute
+                name: k8s.pod.name
+        extract:
+          metadata:
+            - k8s.cluster.name
+            - k8s.namespace.name
+            - k8s.pod.name
+            - k8s.pod.uid
+            - k8s.node.name
+            - k8s.pod.start_time
+            - k8s.deployment.name
+            - k8s.replicaset.name
+            - k8s.replicaset.uid
+            - k8s.daemonset.name
+            - k8s.daemonset.uid
+            - k8s.job.name
+            - k8s.job.uid
+            - k8s.cronjob.name
+            - k8s.statefulset.name
+            - k8s.statefulset.uid
+            - container.image.tag
+            - container.image.name
+      resource:
+        attributes:
+          - key: lightstep.helm_chart
+            value: kube-otel-stack
+            action: insert
+          - key: collector.name
+            value: "${KUBE_POD_NAME}"
+            action: insert
+
+    exporters:
+      elasticsearch:
+        endpoints: ["https://logingest.lightstep.com"]
+        index: "lightstep"
+        headers:
+          "lightstep-access-token": "${LS_ACCESS_TOKEN}"
```

Comment:
I think we want this to be `LS_TOKEN` to match the env var above

Reply:
Thank you, good catch.